service-api/module/Application/config/module.config.php

```diff
@@ -246,6 +246,16 @@
                     ],
                 ],
             ],
+            'remove_case_lpa' => [
+                'type'    => Segment::class,
+                'options' => [
+                    'route'    => '/cases/:uuid/remove-lpa/:lpa',
```

Comment:
FWIW this isn't very RESTful and I'd prefer:
- `PUT /cases/:uuid/lpas/:lpa` to add an LPA to the case
- `DELETE /cases/:uuid/lpas/:lpa` to remove it